config/scenarios.form.yaml

```diff
@@ -118,4 +118,75 @@ main-mds-vanilla-with-grids:
     storage_units: ["li-ion battery", "iron-air battery"]
     H2_network: true
     gas_network: true
-    H2_retrofit: true
\ No newline at end of file
+    H2_retrofit: true
+
+main-calibration-run:
+  foresight: overnight
+  scenario:
+    ll:
+    - v1.0
+    planning_horizons:
+    - 2023
+  enable:
+    final_adjustment: false
+  snapshots:
+    start: "2023-01-01"
+    end: "2024-01-01"
+  co2_budget:
+    2023: 0.378 
+    # 63% from https://commission.europa.eu/news/climate-report-shows-largest-annual-drop-eu-greenhouse-gas-emissions-decades-2024-11-05_en
+    # 60% factored in emission included sector from co2_totals_sector.csv (interpolation between 2015, 2030)
+    # 0.63 * 0.6 = 0.378
+  electricity:
+    max_hours:
+      li-ion battery: [1, 2, 4, 8]
+      vanadium: [10]
+      lair: [12]
+      pair: [24]
+      H2: [168]
+      iron-air battery: [100]
+    powerplants_filter: DateOut >= 2023 and not Fueltype.isin(['Nuclear', 'Hard Coal', 'Lignite'])
+    custom_powerplants: DateIn <= 2023 and DateOut >= 2023
+  atlite:
+    default_cutout: europe-2023-sarah3-era5
+  renewable:
+    onwind:
+      cutout: europe-2023-sarah3-era5
+      #correction_factor: 0.93
+    offwind-ac:
+      cutout: europe-2023-sarah3-era5
+    offwind-dc:
+      cutout: europe-2023-sarah3-era5
+    offwind-float:
+      cutout: europe-2023-sarah3-era5
+    solar:
+      cutout: europe-2023-sarah3-era5
+      correction_factor: 0.854337 #Using ERA5 data requires correction factor for solar.
+    solar-hsat:
+      cutout: europe-2023-sarah3-era5
+    hydro:
+      cutout: europe-2023-sarah3-era5
+      #eia_norm_year: 2021 #The last year available from EIA for hydro is 2021. If not activated, the median value is chosen
+  biomass:
+    year: 2020
+    share_unsustainable_use_retained:
+      2023: 0.8
+    share_sustainable_potential_available:
+      2023: 0.2
+  sector:
+    district_heating:
+      progress:
+        2023: 0.15 #Rough estimate. 70M/448M population uses district heating https://iifiir.org/en/news/state-of-play-of-district-heating-and-cooling-in-europe
```

Comment:
This may be reduced a bit, using [PyPSA-ariadne](https://github.com/PyPSA/pypsa-ariadne/blob/main/config/config.yaml#L292)